### DISCUSSION TITLE
Supported formats

### DISCUSSION THREAD

**Discussion by ZGltYQ** [withcatai/node-llama-cpp#137] [Q&A] [answered]
Does library support only gguf format models ?

**Comment by samlhuillier** [comment]
I think so. It's a wrapper over llama.cpp which itself only supports gguf as far as I'm aware.

**Comment by giladgd** [comment, marked as answer]
`node-llama-cpp` supports the model file formats supported by `llama.cpp`.
AFAIK, right now the only format it supports is GGUF which is a format that was invented specifically for `llama.cpp`
